specs/player_spec.rb

```diff
@@ -0,0 +1,155 @@
+
+describe Scrabble::Player do
+    before do
+        @new_player = Scrabble::Player.new('Tofu')
+    end
+    describe '#initialize' do
+        it 'creates a new instance of player' do
+            Scrabble::Player.new('Tofu').must_be_instance_of Scrabble::Player
+        end
+
+        it 'returns .name as a string' do
+            Scrabble::Player.new('Danielle').name.must_be_kind_of String
+        end
+
+        it 'returns the name of the player correctly' do
+            Scrabble::Player.new('Tofu').name.must_equal 'Tofu'
+        end
+
+        it 'initializes @player_score holds 0' do
+            Scrabble::Player.new('Tofu').total_score.must_equal 0
+        end
+    end
+
+    describe '#plays' do
+        it 'returns .plays as an Array' do
+            @new_player.plays.must_be_kind_of Array
+        end
+        it 'returns all 5 words played in order' do
+            words = %w(zip sqeeze whiz jerky pig)
+            words.each { |word| @new_player.play(word) }
+            @new_player.plays.must_equal words
+        end
+    end
+
+    describe '#play(word)' do
+        it 'returns .play(word) as an Integer' do
+            @new_player.play('whiz').must_be_kind_of Integer
+        end
+
+        it 'returns the correct score of the word' do
+            @new_player.play('zaz').must_equal 21
+        end
+
+        it 'adds the input word to the plays array' do
+            @new_player.play('word')
+            @new_player.plays.must_include 'word'
+        end
+
+        it 'returns the correct score of a 7-letter word' do
+            @new_player.play('squeeze').must_equal 75
+        end
+
+        it 'returns false if the player has already won' do
+            @new_player.play('zzzzzzz')
+            @new_player.play('word').must_equal false
+        end
+
+        it 'wont add to the score if the player has already won' do
+            @new_player.play('zzzzzzz')
+            @new_player.play('jazz')
+            @new_player.total_score.must_equal 120
+        end
+
+        it 'Raises an error if incorrect argument is given' do
+            proc { @new_player.play(7) }.must_raise ArgumentError
+            proc { @new_player.play }.must_raise ArgumentError
+            proc { @new_player.play('word!') }.must_raise ArgumentError
+            proc { @new_player.play('reallylongword') }.must_raise ArgumentError
+        end
+    end
+
+    describe '#total_score' do
+        it 'returns .total_score as an Integer' do
+            %w(zip sqeeze whiz jerky pig).each { |word| @new_player.play(word) }
+            @new_player.total_score.must_be_kind_of Integer
+        end
+        it 'returns sum of scores of played words' do
+            %w(z zzz zz zqaa).each { |word| @new_player.play(word) }
+            @new_player.total_score.must_equal 82
+        end
+    end
+
+    describe '#won?' do
+        it 'returns #won as a Boolean value' do
+            [true, false].must_include @new_player.won?
```

Comment:
This is a nitpick, but I don't think this test is necessary. Since all values in Ruby are either truthy or falsey, explicitly checking whether something is `true` or `false` is a little overkill.